content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+---
+title: APIs
+hasparent: true
+---
+
+Jaeger components implement various APIs for saving or retrieving trace data.
+
+TODO
+
+  * stable
+  * restricted
+  * deprecated
+
+## Span reporting APIs
+
+Agent and Collector are the two components of the Jaeger backend that can receive spans and forward them to storage.
+
+### Thrift over UDP (stable)
+
+The Agent can only receive spans over UDP in Thrift format. The primary API is a UDP packet that contains a Thrift-encoded `Batch` struct defined in [jaeger.thrift][jaeger.thrift] IDL file, located in the [jaeger-idl][jaeger-idl] repository. Most Jaeger Clients use Thrift's `compact` encoding, however some client libraries do not support it (notably, Node.js) and use Thrift's `binary` encoding (sent to  a different UDP port). The Agent's API is defined by [agent.thrift][agent.thrift] IDL file.
+
+For legacy reasons, the Agent also accepts spans in Zipkin format, however, only very old versions of Jaeger clients can send data in that format and it is officially deprecated.
```

Comment:
May be best to have in its own section with the heading having `(deprecated)` to make it clearer.

Reply:
My thinking was that instead we simply don't describe the API, similar to TChannel. I.e. I mention that such thing exist, but provide no details, pointers to IDL.